css/gallery.css

```diff
@@ -0,0 +1,116 @@
+/*class for the section of gallery*/
+.info-gallery {
+    width: 635px;
+    height: 88px;
```

Comment:
When is it correct to use fixed width and height?

Reply:
For this specific case the div is at 365px from each side, since this distance is relatively big and would be hard to handle on various screen sizes it is better to set the width and height, and then just center the div to match the general design feel.

===========

Para esta caso especifico el div esta a 365px del borde, esta distancia es relativamente grande y podria ser problematica de mantener en tamaños cambiantes de pantalla, aqui es mejor dar un ancho y alto, y luego centrar el div para que coincida con la apariencia general del diseño.

a menos que se defina desde reglas, idealmente en el inicio del proyecto que siempre se dara ancho o siempre se escalara por distancia del borde, esto tiene que avaluarse caso por caso.
Ya que no tenemos una definición de este tipo, evaluar cada caso tendria que ser lo que hagamos.

Reply:
by the way, this class is not being used.

Reply:
@Frankruiz90 the class `info-gallery` is never used, only `info-gallery__text`